make_atf.sh

```diff
@@ -36,6 +36,14 @@ find . -type f -and -name '*.pkt' | sort | \
 printf '# This file is %sgenerated by make_atf.sh\n' @
 printf '# in the tcptestsuite distribution.\n'
 
+cat <<EOF
+save_sysctls() {
+    sed -n 's/.*sysctl -w \\([^=]*\\)=.*/\\1/p' < \$1 \\
```

Comment:
From sysctl's man page: "The -w option has been deprecated and is silently ignored".  What were you hoping it would do?

Reply:
This is grabbing the OIDs from the tests, which always use `sysctl -w`.  I suppose someone could remove all the `-w` flags from the tests.  Then I could remove it from here.